cms/jinja2/templates/base.html

```diff
@@ -350,7 +350,7 @@
     onsCookiesBanner({
       'lang': "cy" if LANGUAGE_CODE == "cy" else "en",
       'serviceName': COOKIE_BANNER_SERVICE_NAME,
-      'settingsLinkURL': MANAGE_COOKIE_SETTINGS_URL,
+      'settingsLinkUrl': MANAGE_COOKIE_SETTINGS_URL,
```

Comment:
missed in https://github.com/ONSdigital/dis-wagtail/pull/34/files#diff-bd7285587bee7451cbc78ed59b209035ad01327b8f0dcbfe38341cea13801682L120